src/components/error/FrontEnd/FrontEndError.css

```diff
@@ -0,0 +1,59 @@
+.dash-fe-errors {
+}
+
+.dash-fe-error-top {
+    display: flex;
+    align-items: center;
+}
+.dash-fe-error__icon-error {
```

Comment:
Or maybe it's an icon-error? :)

Reply:
That would be the case if there was a `dash-fe-error__icon` base class, but since there isn't, it doesn't make sense to use a modifier (`--`) here I think.